onnxscript/function_libs/torch_aten/graph_building.py

```diff
@@ -397,7 +397,19 @@ def _add_torchscript_op_call(
         graph_inputs = []
         assert isinstance(unwrapped_inputs, Sequence)
         for input in unwrapped_inputs:
-            if not isinstance(input, torch.Value):
+            if isinstance(input, Sequence) and all(
+                isinstance(elem, torch.Value) for elem in input
+            ):
+                # If all elements in the Sequence are torch.Values we know it
+                # should be a Sequence input in ONNX.
+                input_sequence = _create_op_call_in_torch_graph(
+                    self._torch_graph,
+                    "onnx::SequenceConstruct",
```

Comment:
I think it would make more sense to move this to PyTorch side, like we did on op.Concat? We should process all the list related stuff in the same place. Especially, fx.node might not always be tensors.

Reply:
I think this is different because we annotate the type as `Sequence[Tensor]`, whereas for the Concat case we annotated as `INT64`. We can chat more

Reply:
Well, the current type system we use shows that the functions take a (python) Sequence. If we do SequenceConstruct outside, the input will not be a Sequence. However, we can also extend the return value of SequenceConstruct to be a proper Sequence so that we can do what you suggested